packages/cnb_app_lifecycle/spec

```diff
@@ -0,0 +1,373 @@
+---
+name: cnb_app_lifecycle
+
+dependencies:
+  - golang-1.22-linux
+  - healthcheck
+  - diego-sshd
+
+files:
+  - code.cloudfoundry.org/go.mod
+  - code.cloudfoundry.org/go.sum
```

Comment:
missing:
```
 - code.cloudfoundry.org/vendor/modules.txt
 ```

Reply:
We added the change and resolved the conflicts.